### DISCUSSION TITLE
Check update from v0.107.0-b.17

### DISCUSSION THREAD

**Discussion by alexdelprete** [AdguardTeam/AdGuardHome#3985] [Q&A] [answered]
I'm using v0.107.0-b.17 on OPNsense (FreeBSD). Yesterday I updated from b16 to b17, then after some hours, 0.107 final was announced, but if I check update it tells me I have last version installed. Can I force the update in some way?

Thanks.

**Comment by ainar-g** [comment, marked as answer]
Hello.  Release channels are only updated when a new version is uploaded to the corresponding channel.  That is, if you have a beta, you'll have an update notification when a new beta is out.

You can either:
 *  Stop AGH, download the binary from the releases pages, replace the old binary with the new one, and restart AGH.
 *  Just do nothing, if you are okay with living on the beta channel.  `v0.107.0-b.17` is built from the same commit as the final release.

There is also a feature request about switching the release channel from the UI: #2420.

**Reply by alexdelprete** [reply]
Thanks a lot for the explanation.

Actually the installation on OPNsense is made through a plugin, so I'll have to ask the maintainer to update it since I can't switch channel from the UI. Anyway since it's the same version of the code, there's no rush. :)
